lib/faker/default/internet.rb

```
@@ -174,37 +174,46 @@ def password(legacy_min_length = NOT_GIVEN, legacy_max_length = NOT_GIVEN, legac
           keywords << :special_characters if legacy_special_characters != NOT_GIVEN
         end
 
-        raise ArgumentError, 'Password of length 1 can not have both mixed case and special characters' if min_length <= 1 && mix_case && special_characters
+        use_min_length_as_max_length = (max_length - min_length).negative?
```

probably should just raise an error if max is less then min then silently ignoring the error

---

i think that makes sense... just wasn't sure if i was ok to change the existing behaviour which (if i'm understanding it correctly) does just allow for max to be less than min. i'll update to raise an error instead.